tests/x11/libqt5_qtbase.pm

```diff
@@ -0,0 +1,72 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Tests for libqt5-qtbase
+# Maintainer: Timo Jyrinki <tjyrinki@suse.de>
+
+use base 'x11test';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use x11utils 'turn_off_gnome_screensaver';
+use version_utils 'is_sle';
+use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
+
+sub run {
+    my $self = shift;
+    turn_off_gnome_screensaver if check_var('DESKTOP', 'gnome');
```

Comment:
`turn_off_gnome_screensaver` uses `script_run` and so needs an `xterm`. Currently it works by chance as it's the first test switching to x11.

Reply:
Also, why is this required and why on GNOME only?

Reply:
Not required, it was just a precaution in case the production servers would be much slower. However, there's also KDE being tested and there was no clear reson to have it so removed. ensure_unlocked_desktop sounds like a generic thing good to have, so adding that.